### ISSUE TITLE
Estonia flag colors are not the correct one

### ISSUE THREAD

**Issue by pixelastic** [tao-guo/country-flags#1]
Seems like your estonian flag is not using the correct colors.

![image](https://cloud.githubusercontent.com/assets/283419/25906812/f8c0dcbe-35a5-11e7-81fb-7c6d45dea5ac.png)
![image](https://cloud.githubusercontent.com/assets/283419/25906820/fd96a278-35a5-11e7-8673-16e34b2db8b2.png)

Great job by the way!

**Comment by tao-guo** [comment]
Yeah, I just create this project for learning purpose before realizing your awesome one div css-flag project.  I have only checked the specification for  several countries (China, US, UK, South Korea). I guess I could make this project retired now :smiling_imp:
